responses/14_import-course.md

```diff
@@ -0,0 +1,42 @@
+# Import your course
+
+Woohoo! You could keep going, and we will later - but now your course can actually be _tested_. Let's talk about that.
+
+### Create and test draft courses
+
+The first time you create a course on Learning Lab, the course will be [a draft](https://lab.github.com/docs/testing).  While in draft, you can test your courses by going to the course dashboard, and clicking on View course page next to the course title.
+
+You can continue to [work on your course](https://lab.github.com/docs/testing#test-your-course) while it is in draft -- it will only be visible to you and other administrators of your repo.
+
+### Choosing public vs private courses
+
+If you'd like, you can sign up for private course options as an organization. Contact learninglab@github.com to learn more.
+
+### Publish a course
+
+When you're ready to [publish your course](https://lab.github.com/docs/publishing), just go to your course's administration panel and click the button! Then, other users can register for and take your course.
+
+### Maintaining launched courses
+
+Once a course is published, you can continue to maintain it by using drafts. When you create a branch and a pull request in an existing course, Learning Lab will automatically create a draft deployment of that pull request so you can test. There will be links directly in your pull request, so you should be able to find it easily. These drafts are only accessible to the owner, or if the owner is an organization, those with administrator access.
+
+### Using the Course Builder
+
+This Learning Lab course has walked you through course creation from scratch - but you have other options. You can start with a boilerplate and create courses directly through the Course Builder UI.
+
+## Step 14: Import the course
+
+Importing the course will create the course as a **draft**. While in draft, you can test your courses by going to the course dashboard, and clicking on **View course** page next to the course title.
```

Comment:
Good point. I think the reason I had it there was because there was so much other information in the body of this response, I wanted to remind the learners of the most important things for this step. That made me think that maybe the rest of the information doesn't truly belong in this step. I have deleted the duplicated bits, and moved the parts about publishing, private and public, and maintaining, to the last step in 3e95a51.